playbooks/edx_sandbox.yml

```diff
@@ -1,32 +1,46 @@
 ---
+- name: Provision sandbox
+  hosts: localhost
+  gather_facts: false
+  vars:
+    terminate_instance: true
+  roles:
+    - launch_ec2
 
 # Example sandbox configuration
 # for single server community
 # installs
 
 - name: Configure instance(s)
-  hosts: all
+  hosts: launched
   sudo: True
   gather_facts: True
   vars:
     migrate_db: "yes"
     openid_workaround: True
     EDXAPP_LMS_NGINX_PORT: '80'
     edx_platform_version: 'master'
+    # edx_platform_version: 'dcadams/merge_upstream_hotfix-2015-11-10_2'
+    edxapp_theme_version: 'master'
+    edxapp_use_custom_theme: true
+    edxapp_theme_name: 'lagunita'
+    edxapp_theme_source_repo: 'https://github.com/Stanford-Online/lagunita-theme.git'
```

Comment:
We probably don't want this.

Try to pull from CLI, fallback to defaults/main.yml.